### ISSUE TITLE
Sorting History Manga

### ISSUE THREAD

**Issue by Samarth170102** [KotatsuApp/Kotatsu#428]
### Describe your suggested feature

In shelf page, when I click on the see all button on history page in three dots section there is one option "sort" click on that it will sort alphabetically, or incomplete first, completed first. It will make easier to read the mangas which are incomplete.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.

**Comment by Koitharu** [comment]
Good idea, I guess it might be a great feature to sort history by reading progress also

**Comment by Samarth170102** [comment]
Yes, excellent idea 👍

**Comment by Koitharu** [comment]
Implemented in v6.0

**Comment by Samarth170102** [comment]
I tried beta version, it is working fine, one suggestion when I click on
sorting order as progess, if it has also reverse the order it would be also
good, I mean it sort by progress to lower to higher, in current update it
is higher to lower

**Comment by Samarth170102** [comment]
There is bug in unstable version,
The web comic mode is not working, it do not force to non-web comic to show
as webcomic reader mode
